nixos/modules/tasks/auto-upgrade.nix

```diff
@@ -96,6 +96,30 @@ in {
         '';
       };
 
+      rebootWindow = mkOption {
+        description = ''
+          Define a lower and upper time value (in HH:MM format) which
+          constitute a time window during which reboots are allowed.
```

Comment:
```suggestion
          Define a lower and upper time value (in HH:MM format) which
          constitute a time window during which reboots are allowed after an upgrade.
          This option only has an effect when `allowReboots` is enabled.
          The default value of `null` means that reboots are allowed all the time.
```

Reply:
Also, the `allowReboots` option should have a reference to the `rebootWindow` option like

> See `rebootWindow` for configuring the times at which a reboot is allowed.